SPECS-CROSS/gcc-bootstrap_cross/gcc-bootstrap3_cross.spec

```diff
@@ -0,0 +1,442 @@
+%global debug_package %{nil}
+%global set_build_flags %{nil}
+
+# Globals which should be in a macro file.
+# These should be set programatically in the future.
+%global _host_arch      x86_64
+%global _target_arch    aarch64
+
+%global _tuple          %{_target_arch}-%{_vendor}-linux-gnu
+%global _cross_name     %{_target_arch}-%{_vendor}-linux-gnu
+
+# Folders which should be in our macro file
+%global _opt                /opt/
+%global _crossdir           /opt/cross/
+
+# Generally we include '/usr' in most paths.
+# Can we also use '/usr' for our paths? This will bring us in line with the
+# %%configure macro which sets these.
+%global _bindir            /bin
+%global _sbindir           /sbin
+%global _libdir            /lib
+%global _lib64dir          /lib64
+%global _libexecdir        /libexec
+%global _datadir           /share
+%global _docdir            /share/doc
+%global _includedir        /include
+%global _infodir           /share/info
+%global _mandir            /share/man
+%global _oldincludedir     /include
+
+# If we want our cross compile aware packges to also support native, we
+# need logic to switch modes something like this:
+%if %{_target_arch} != %{_host_arch}
+%global _cross_prefix       %{_crossdir}%{_tuple}/
+%global _cross_sysroot      %{_crossdir}%{_tuple}/sysroot/
+%global _cross_includedir   /usr/%{_host}/%{_tuple}/include/
+%global _cross_infodir      %{_crossdir}%{_tuple}/share/info
+%global _cross_bindir       %{_tuple}/bin
+%global _cross_libdir       %{_tuple}/lib
+%global _tuple_name         %{_tuple}-
+
+%global __strip %{_cross_prefix}%{_bindir}/%{_tuple_name}strip
+%global __objdump %{_cross_prefix}%{_bindir}/%{_tuple_name}objdump
+%else
+%global _cross_prefix       %{nil}
+%global _cross_sysroot      %{nil}
+%global _cross_includedir   %{_includedir}
+%global _cross_infodir      %{_infodir}
+%global _cross_bindir       %{_bindir}
+%global _cross_libdir       %{_libdir}
+%global _tuple_name         %{nil}
+%endif
+
+Summary:        Contains the GNU compiler collection
+Name:           %{_cross_name}-gcc-bootstrap3
+Version:        9.1.0
+Release:        11%{?dist}
+License:        GPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Tools
+URL:            https://gcc.gnu.org/
+Source0:        https://ftp.gnu.org/gnu/gcc/%{name}-%{version}/gcc-%{version}.tar.xz
+Source1:        https://ftp.gnu.org/gnu/mpfr/mpfr-4.0.1.tar.gz
+Source2:        http://ftp.gnu.org/gnu/gmp/gmp-6.1.2.tar.xz
+Source3:        https://ftp.gnu.org/gnu/mpc/mpc-1.1.0.tar.gz
+Patch0:         090_all_pr55930-dependency-tracking.patch
+# Only applies to the Power9 ISA
+Patch1:         CVE-2019-15847.nopatch
+BuildRequires:  %{_cross_name}-binutils
+BuildRequires:  %{_cross_name}-kernel-headers
+BuildRequires:  %{_cross_name}-glibc-bootstrap2
+AutoReqProv:    no
+Conflicts:      %{_cross_name}-gcc
+Conflicts:      %{_cross_name}-gcc-bootstrap2
+Conflicts:      %{_cross_name}-gcc-bootstrap3
```

Comment:
Why is there a `Conflicts` on the package itself?

Reply:
Good catch, should be 1 & 2.